cmd/minikube/cmd/start_flags.go

```diff
@@ -276,6 +276,15 @@ func generateClusterConfig(cmd *cobra.Command, existing *config.ClusterConfig, k
 			glog.Errorf("Found deprecated --enable-default-cni flag, setting --cni=bridge")
 			chosenCNI = "bridge"
 		}
+		// networkPlugin cni deprecation warning
+		chosenNetworkPlugin := viper.GetString(networkPlugin)
+		if chosenNetworkPlugin == "cni" {
+			glog.Errorf("Found deprecated --network-plugin=cni flag (new flag: --cni), setting --cni=auto if not already set")
```

Comment:
thank you for doing this PR
I recommend using out.Warnning here so it looks better in the UI, and you mind please paste the before and after this PR output in the description ?